app/models/ability.rb

```diff
@@ -250,9 +249,10 @@ def signed_in_with_cfp_role(user)
       (Conference.with_role(:cfp, user).pluck(:id).include? role.resource_id)
     end
 
-    can [:index, :revert_object, :revert_attribute], PaperTrail::Version, item_type: 'Event', conference_id: conf_ids_for_cfp
-    can [:index, :revert_object, :revert_attribute], PaperTrail::Version, item_type: 'Vote', conference_id: conf_ids_for_cfp
-    can [:index, :revert_object, :revert_attribute], PaperTrail::Version do |version|
+    can [:index, :revert_object, :revert_attribute], PaperTrail::Version,
+        item_type: %w(Event EventType Track DifficultyLevel EmailSettings Room Cfp Program Comment), conference_id: conf_ids_for_cfp
+    can [:index, :revert_object, :revert_attribute], PaperTrail::Version,
+        ["item_type = 'Commercial' AND conference_id IN (?) AND (object LIKE '%Event%' OR object_changes LIKE '%Event%')", conf_ids_for_cfp] do |version|
```

Comment:
This allows users with role cfp to access commercials of events (only of events, not of conferences or venues). This is the SQL equivalent for that, and we need it, in the case of a block definition of ability, for accessible_by to work
https://github.com/openSUSE/osem/pull/1124/files#r75135590
https://github.com/CanCanCommunity/cancancan/wiki/Defining-Abilities-with-Blocks#fetching-records